src/icrc1_auction_api.mo

```diff
@@ -42,8 +42,11 @@ actor class Icrc1AuctionAPI(quoteLedger_ : ?Principal, adminPrincipal_ : ?Princi
   let adminsMap = RBTree.RBTree<Principal, ()>(Principal.compare);
   adminsMap.unshare(stableAdminsMap);
 
-  stable var assetsDataV2 : Vec.Vector<{ ledgerPrincipal : Principal; minAskVolume : Nat; handler : TokenHandler.StableData; decimals : Nat }> = Vec.new();
-  stable var assetsDataV3 : Vec.Vector<StableAssetInfo> = Vec.map<{ ledgerPrincipal : Principal; minAskVolume : Nat; handler : TokenHandler.StableData; decimals : Nat }, StableAssetInfo>(assetsDataV2, func(ad) = { ad with symbol = "" });
+  stable var assetsDataV3 : Vec.Vector<{ ledgerPrincipal : Principal; minAskVolume : Nat; handler : (((RBTree.Tree<Principal, { var value : Nat; var lock : Bool }>, Nat, Nat, Nat), Nat, Nat, Nat, Nat), ([(Principal, Int)], Int, Int)); symbol : Text; decimals : Nat }> = Vec.new();
+  stable var assetsDataV4 : Vec.Vector<StableAssetInfo> = Vec.map<{ ledgerPrincipal : Principal; minAskVolume : Nat; handler : (((RBTree.Tree<Principal, { var value : Nat; var lock : Bool }>, Nat, Nat, Nat), Nat, Nat, Nat, Nat), ([(Principal, Int)], Int, Int)); symbol : Text; decimals : Nat }, StableAssetInfo>(
+    assetsDataV3,
+    func(ad) = { ad with handler = ad.handler.1 },
+  );
```

Comment:
What's the difference here between types V3 and V4?

Reply:
The difference is the type of TokenHandler's stable data

Reply:
Yes, I just couldn't spot the difference :)

Reply:
Old TokenHandler data type is 
```
public type StableData = (
    AccountManager.StableData, // account manager
    CreditRegistry.StableData, // credit registry
  );
```

New one is simply `CreditRegistry.StableData`